gym/vector/vector_env.py

```diff
@@ -69,7 +69,7 @@ def reset_wait(
         seed: Optional[Union[int, List[int]]] = None,
         return_info: bool = False,
         options: Optional[dict] = None,
-    ):
+    ) -> np.ndarray:
```

Comment:
Don't vector envs have the same (obs, info) semantic as general envs?

Reply:
I thought it was the cause of some issues but it doesn't raise an error anymore so will revert